package.json

```diff
@@ -84,7 +84,7 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.20.0 <13.0.0"
+    "node": ">=10.20.0 <15.0.0"
```

Comment:
My thinking may be flawed, but on this line in `package.json`, I think that we should only "prevent" (warn against) usage of versions that we know not to work. So I'd recommend:
```
    "node": ">=10.20.0"
```
At the same time, this PR could amend the [INSTALL-ADVANCED.md](https://github.com/balena-io/balena-cli/blob/master/INSTALL-ADVANCED.md#npm-installation):

**From:** `Node.js version 10 (min 10.20.0) or 12 (version 14 is not yet fully supported)`
**To:** `Node.js version 10 (min 10.20.0), 12 or 14 (other versions may also work, but are not well tested)`

The GUI installers and the standalone zip package bundle their own Node.js (currently v10 or v12), so only users who choose the _not-recommended_ advanced installation option -- npm install -- are affected by this change in `package.json`, and even then the documentation warns them that only versions 10, 12 and 14 are tested. So if they choose to use Node.js v16, v25... and find bugs and report them, they'd effectively be testing Node.js for us, and saving us the trouble. :-)

Reply:
I like this idea. @srlowe what do you think?